.circleci/config.yml

```diff
@@ -633,13 +635,19 @@ jobs:
       - run: *yarn
 
       - run:
-          name: Publish React Native Package
+          name: Authenticate with npm
+          command: echo "//registry.npmjs.org/:_authToken=${CIRCLE_NPM_TOKEN}" > ~/.npmrc
+
+      - run:
+          name: Authenticate git user
```

Comment:
Is this even necessary? We are only using git in the publish script to read the git tags, there is no write action.